doc/guide.rst

```diff
@@ -0,0 +1,179 @@
+.. _guide_label:
+
+Guide to Simpleline
+===================
+
+Simpleline is the text user interface framework completely written in Python 3 with a possibility to
+have :ref:`non-python event loop <event_loops_label>`. With exception of optional event loops
+Simpleline doesn't have almost any dependency to external libraries.
+
+This UI is simple and easy to use. It is designed to be used with line-based machines and tools
+(e.g. serial console) so that every new line is appended to the bottom of the screen.
+Printed lines are never rewritten!
+
+Basic components
+----------------
+
+For every application the following parts are always required and a developer of an application
```

Comment:
`every application` -> `every application,`

I'm not sure if it's necessary to include the last piece of the sentence, from "and a developer..." to the end. Is that not implied?